### PR TITLE
Bob and bobby/7768 fix checkbox multiselect bug

### LINKED ISSUE #7768: [BUG] Symptoms checkbox unselectable when multiple patients in test queue
### Description
A user cannot select symptoms for a patient when the test queue has multiple patients with the same disease and symptoms that are able to be selected.

### Steps to reproduce
1. Go to "Conduct test"
2. Add a patient to the queue
3. Add another patient to the queue
4. For both patients, test the same device
5. For both patients, select "Yes" for the "Is the patient currently experiencing any symptoms?" question
6. Try to select the symptoms for the patient at the bottom of the queue
7. Notice the first patient in the queue's symptoms checkboxes are being selected instead

https://github.com/CDCgov/prime-simplereport/assets/20211771/0caf12b1-9544-48da-8649-579a07dbec75

### Expected behavior
A user can select symptoms for each patient in the test queue regardless of whether there are other test cards in the queue testing for the same disease and symptoms.

### Additional context
This issue may be similar to the root issue for #7730

### PR DESCRIPTION
**Pull request by fzhao99**
# FRONTEND PULL REQUEST

## Related Issue

- Fixes #7768 

## Changes Proposed

- Fixes the markup in the checkbox component to be unique across values
- Makes name across the facility management forms unique
- Adds a regression test

## Testing
deployed on dev 2: verify that checkboxes across cards doesn't break

## Screenshots / Demos
https://github.com/CDCgov/prime-simplereport/assets/29645040/1400b0b9-caa6-4ea4-a348-86092cbce156

### CODE REVIEW

File: frontend/src/app/commonComponents/Checkboxes.tsx [hunk 1]
```diff
@@ -87,7 +87,7 @@ const Checkboxes = (props: Props) => {
   return (
     <div
       className={classnames(
-        "usa-form-group",
+        "usa-form-group padding-0",
```
Comment (fzhao99) [inline frontend/src/app/commonComponents/Checkboxes.tsx:90]: noticed some extra padding in the AOE form so adding this to fix

Screenshot 2024-06-05 at 3 59 32 PM

File: frontend/src/app/commonComponents/Checkboxes.tsx [hunk 2]
```diff
@@ -128,11 +128,12 @@ const CheckboxesFragment = (props: FragmentProps) => {
   const { boxes, name, inputRef, onChange } = props;
 
   return boxes.map(({ value, label, disabled, checked, ...inputProps }) => (
-    <div className="usa-checkbox" key={value}>
+    <div className="usa-checkbox" key={`${name}-${value}`}>
       <input
         className="usa-checkbox__input"
         checked={checked}
-        id={`symptom-${value}`}
+        id={`${name}-${value}`}
```
Comment (fzhao99) [inline frontend/src/app/commonComponents/Checkboxes.tsx:135]: core issue causing the bug was that `symptom-${value}` wasn't a unique ID when there were multiple cards for the same disease. Assuming here that name is a unique identifier so adding it fixed it at least for the test queue.

File: frontend/src/app/commonComponents/Checkboxes.tsx (hunk 1 above)
Comment (emyl3) [reply frontend/src/app/commonComponents/Checkboxes.tsx:90]: OO thank you for fixing this!

**Review by mpbrown — APPROVED** [review]
LGTM on dev3!

**Review by emyl3 — APPROVED** [review]
LGTM! deployed and tested on dev3! Nice work @fzhao99 and @bobbywells52 😸

**Pull request merged**